packages/aws-cdk-lib/aws-rds/lib/cluster.ts

```diff
@@ -492,6 +502,25 @@ export enum InstanceUpdateBehaviour {
 /**
  * The scalability mode of the Aurora DB cluster.
  */
+export enum ClusterScalabilityType {
```

Comment:
Would have done the following:
`export import ClusterScailabilityType = ClusterScalabilityType`

which would allow updates to `ClusterScalabilityType` to also be taken on by the misspelled enum. But that fails the public API regression checker (and the build). So both will need to be updated if any changes are made to the enum.